core/state/statedb_test.go

```diff
@@ -1051,7 +1051,7 @@ func TestFlushOrderDataLoss(t *testing.T) {
 		t.Fatalf("failed to commit state trie: %v", err)
 	}
 	triedb.Reference(root, common.Hash{})
-	if err := triedb.Cap(1024); err != nil {
+	if err := triedb.Cap(128); err != nil {
```

Comment:
what's the reasone behind lowering this limit?

Reply:
zkTrie nodes are smaller, so we need a lower limit here to trigger a flush to the disk.